library/src/options.cxx

```diff
@@ -192,6 +192,7 @@ F3D_DECL_TYPE(int);
 F3D_DECL_TYPE(double);
 F3D_DECL_TYPE(f3d::ratio_t);
 F3D_DECL_TYPE(std::string);
+F3D_DECL_TYPE_INTERNAL(color_t);
```

Comment:
maybe rename this define ?

Reply:
What about?

`F3D_DECL_TYPE_INTERNAL` -> `F3D_DECL_TYPE`
`F3D_DECL_TYPE` -> `F3D_DECL_TYPE_WITH_VEC`